quipucords/quipucords/settings.py

```diff
@@ -169,9 +116,7 @@ def create_random_key():
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [
-            os.path.join(os.path.dirname(__file__), "templates").replace("\\", "/"),
-        ],
+        "DIRS": [os.path.join(os.path.dirname(__file__), "templates")],
```

Comment:
ahem, have you heard the gospel of the std lib `pathlib`, young man?
lemme tell ya

```suggestion
        "DIRS": [Path(__file__).parent /  "templates"],
```

(`pathlib` is probably my favorite lib in standard library XD)

Reply:
LOL yeah, I _definitely_ noticed how inconsistently strings and Path objects were being treated in the settings file, some using `/` some using `join`, and a whole mess of different variables that probably could be simplified and consolidated, but I mentally compartmentalized all that into the

> Perhaps we should return and scrutinize all of the settings more closely again in the not-too-distant future.

comment I put at the top of the PR. 😅

Reply:
I manually added your suggestion to a commit on this branch instead of accepting it in GitHub because an extra space in `/  "` makes black unhappy.